### PR TITLE
Reorganize how normalized instance attributes are handled

### PR DESCRIPTION
**Pull request by ptrgags**
Fixes https://github.com/CesiumGS/cesium/issues/10766

As of the latest `main`, instance feature IDs weren't working because they were incorrectly being dequantized in `GltVertexBufferLoader`. 

As described in https://github.com/CesiumGS/cesium/issues/10766#issuecomment-1239843640, we thought it would be better to not dequantize in the loader at all, only dequantize when absolutely necessary. It turns out there's only one such case where we need to dequantize on the CPU: `ROTATION` attributes with `normalize: true`. Translations and scales are not quantized according to the table in the [`EXT_mesh_gpu_instancing` spec](https://github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Vendor/EXT_mesh_gpu_instancing#extending-nodes-with-instance-attributes).

To summarize, this PR:

* Removes CPU dequantization code from `GltfVertexBufferLoader` and `GltfLoader`
* Only dequantizes rotations in `InstancingPipelineStage`

EDIT: Whoops forgot to add Sandcastle links:

* [Sandcastle from the issue](http://localhost:8080/Apps/Sandcastle/index.html#c=jVcLT9tIEP4re7nqzhFhE15qGwK6OwoFHZSqSdFVTVXW9jjZ1t61vOsARfnvN/uwY4LTOwkI9n7z2JlvHomkUJosONxBQY6IgDtyAoqXGb2x74JpJ7LPJ1JoxgUU0073cCqmIrKSUam0zMZzFq/LnzROgsepICTls7nmYnYlY0iHFe6y+ZZ+fHd5MekZdFKwWQZCOw0TuNdDcmsOFpLH9ekV+hSc+YcLkZeaJMp+9ggXEh+jH9nXzOi+YhoKzlKS+X+6xLpF6hc05klSKsCLLCDaC5JUMh14fTQBpssCLmJFOV6diQjOqldfB13SJzsDOjDBIWRp/qC3Sxerfp+cFIgFouf4y1NQgA4yEZNMLoBwTbQkjERQmCCTXCquuRQEf4zELJUhVCH34sZJmyKqIhBA84JnKLQARVkcB8aBZjbsx96biRMO/M3LAhNxO9c6H/b7qYxYOpdKD18NXu30Z+nkrP82Ly/8ZdUVaBYzzfreA/pNSXHbc5piCMvZeC7v/pKliDGfNzItMxgSXZTgMU2y2Ch1p8IEqNJXpang93g77/mkYEIlssgUBab0O1no+cd8Is/4PcSY+QzsXat7skLjf0zs0aSQ2RuYFQAq2H55QHcOdvd2D3pk7zV9vf9ysN8jA2/fxVUmiQtrI2znwMxd3nMdzT8wMXtiCz2dUy0/IARdDHZ36UG3t/l8ewXYR6I40z6FP6TMJjKomeF8WScPE+R8cnVJIAVDd6QG0+SOpylRUCCNmDJssRJaylTz3EFirvKUPdhT4mmM1WGCygzNpiIpRWQJF1lLEycd+ArxvPMqj0gso9I4QB361LmDrSLmC9ceSMXNqGoblOU5iPhkztM48Koc0j9QpR9SoCGLvs8Kw6ETmUrTU6adMMWX004Lui4UA2OhQs5paEWmkGiLGrQea5n/5DTHikIaWMTOfn7fBvo87eSSC+wk27DAcKhp54sVEFLABoEQS+77tuI/oAInXG+bmKECL1MApktUovhqWTeCOiFrWatoY8rR9Q9sC547Xjmm3/UiJ2PhTDfACnXCqhM1KKJQa2XKof7pefinHpnrLD1xNjx7nobSc9HnVW7Iq8/W7YtHb2GZ399uTFsN+/QMxgVyz178qOmaC2NbbSnsPphpWzeevOpprZhIhWayrILla6pHkLY4B1cHmusUnpWXHxvn6E9gEb1Kgw8ZT0jwi+8iMSRYP3FQIeqp5YlxG7yT5MWj1bPsjsLi+HY1ghxR8kLmOFke/gbsAUeVLToD/d6f4FALXDW2mW7Kt9v3CA6KYKT+wx0c3ysn6qlqUSk2YJMmk9RReFzpQYU8Xg5H/fB4pdEYCowAR/TgED9GT26KJBIzPceDra3a6WcBQdmm0Gf+5bANecPSEtpj1wxP1wvbO2zhJX4TocoP3d8Xjw3kkgzJ6oVVv1yLlg+w0dXg61tLLKSrDL9BVA0BbP0ytJ02xtbLyNjsBL/mPPo+JBeJWTC4ctQlzVXgrCIuUlY4lDcLMbX4azwp7rhCzO+G3ZYRv6/han47H7zWwJiH+Cec9oB1StV2njFHedUma1aWVKuYTNpWHQ9fMbvW8D9s+hNnqJEBuxDh0DVlPscdLrUlX41izKdpFlbMdJI7E9lqLGfSrJcYPLP2xW4DLPO42gxTFkJKsD9x/bBBEOWK9iRWY6Fy6ckiM7b9cZyzCE7NdDp3oODJChkxsWDKcNjroLiE2OX3T5veoM5zYPzPVj3elKFvuWZTt93dN3ZfRy5b6zsrvq1VUVwP3vtx022IJnXKW9nVbPfW9tZRS5uddqoodVbNtmHEjxk00uYOvX8G/bQJ+mChPx+SDXeNE8vepjRNHnKgV9cfx6dfr65vTruHnV5nZEffsWMvIX/wLMed2KzyAaV9DRnOV9yB+2GJ8cEdTamqK436TdERbmvYi49avuGRKGVK4UlSpunYrifHoz7in4niNySzGV0jK3GqG9h85/jSvaSUjvr42C5pYhCyYk3zvw) -- it should look like a gradient in this branch:
image
* [Sandcastle to try the test model](http://localhost:8080/Apps/Sandcastle/index.html#c=tVdtb9s2EP4rnD/JgEvZTtxunhMscda12NJ1sdsNmIeCoU4WEYoUSCovLfLfdyIlW34BBiNzECDk8XjP89yR1IVrZR25F/AAhpwRBQ9kClaUOf3sbdGiw/18qpVjQoFZdHrk20IRIlSqL/XjmKRMWuhVJgsSuBNavVeJ4Mxps7GasUQ/2DFxpmwsupTJhRI5c7CyP3d/XKiFSkvlYxFuAJevdQIyKo3skQzEMnPdQCNwp6CccAIsNZDre7iQMgphCOFeY6Gt8OHOGoVTZhyOmDqhqdH5FSwNgI2qLYS8GgxPaP/N6enrwQ+9YDo9pf1R/+RN/3VtCDyqcQXVAGXAEqGWa5xr5jLq9A2ambLR4GTUdi+E4xk691s2o6XcMmXFVn3eBZiP1fYb9I9q3F6I2PNB2kDaCEwS28zB3CClVJvc0mwr4B8lZt0odK9T0mSwUV+YRvoaxJfhCeNvl4UlSfQtbFQsx2JXldyMO95GaDEetyf1cl6diDGpoxKMKNph0UHgySrzj+IR5Ex8RdDB8Pv1KnusVmecSVwZ9vGnXnv2f59D8mohzjB+B8nPjb4gFD2eK/l1houKncXVv6udNTEHj25MFp336MIUh4REHzDhTCKjhNzooKmL96pWrcI1wqvT3ICou1bZvg2NjWD4eFYAt/EVcyz2qzZeyvnbV0Paj1fQfwqXrcEbbO/4X050KV26Iln9DEbNOGRqlbjeHvUXwnDDUvdSmZTG+DtjeSHBa82D1nCeEaUefcEhUr7dYDzCGtP+AayvjFZwXMoeoj3eYT0YHUb6F6NLlZDPkAkuj8Y+oNQgm7MdBYewf6cdwdqRSyal1uq4ya9BNme7BTj02FwLeUfmpuR3x6Vf4XiY7fmLKjC7E0rh0zTNGD56zn/yjymCNfn/gsMXMf+kpHAEm5JjMcbQHmM12PMqHvrCMK7JVOeFAWsx657Msfh7tDXYnmOzpeYQKb/O/xq2lRx4g0VKou/qtuQtyioNXIEL/SS1ZVFo4+wls8JG9TfZclDQbYUg5EEo7DEpftKNa2Wkysk8E5bcGuxAsdVNNFii8KmpAxMfmHxS4h6MZRLbmZWOSh+SsYvOOmCTDkzF//ICVbnbtbz8JVo3Hce9FSucjclhPcM/vpOcMZVwZh1+RrBnnGstb5m5BlVGdXtVbez0OhPrniScN+F+ErmvI/Z/EZJ0gCRRhI1v8VyDo9zaBnESt7dOEnFPRHK25x8dwiWzFlfSUvoGctE5n8Tov7NVat87/46HR7Knyi0bnP8WjJTSSYzT/TtdELgV+V8) -- NOTE: Since instancing bounding spheres use the translation min/max and don't take into account the rotations (that would be more expensive), the top-most box sticks out a little too much and gets clipped with some camera views.

image
2022-09-08_BoundingVolumeNotQuiteRight

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @ptrgags!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/fix-instancing-feature-ids/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

File: Specs/Scene/Model/InstancingPipelineStageSpec.js [hunk 1]
```diff
@@ -425,6 +427,58 @@ describe(
       });
     });
 
+    it("dequantizes normalized rotations", function () {
```
Comment (ptrgags) [inline Specs/Scene/Model/InstancingPipelineStageSpec.js:430]: Looks like this test is failing in Travis:

`DeveloperError: IndexDatatype.UNSIGNED_INT requires OES_element_index_uint, which is not supported on this system.  Check context.elementIndexUint.`

As with other instancing-related specs, I probably just need to skip this test in Travis

**Comment by ptrgags** [comment]
Okay, this is now ready for review. @sanjeetsuhag could you review when you get a chance?

**Review by sanjeetsuhag — APPROVED** [review]
Everything looks good. Seems to be the correct fix. Thanks @ptrgags!

**Pull request merged**